#########################################################
# You must re-run configure.py after changing this file #
#########################################################

# Paths relative to src/

#extab.s:
  #extab_:      [0x80006720, 0x80012840]
  #extabindex_: [0x80012840, 0x80020b00]

#ctors_dtors.s:
    #.ctors: [0x80480080, 0x80480160]
    #.dtors: [0x80480160, 0x80480180]

#PowerPC_EABI_Support/Runtime/global_destructor_chain.c:
  #.text: [0x802b1f14, 0x802b1f74]
  #.sbss: [0x8050e0c0, 0x8050e0c8]

#PowerPC_EABI_Support/Runtime/NMWException.cp:
  #.text: [0x802b1f74, 0x802b24f4]
  #.sdata: [0x8050cf98, 0x8050cfa0]
  #extab_: [0x8001196c, 0x800119ac]
  #extabindex_: [0x8001fdb8, 0x8001fde8]

#PowerPC_EABI_Support/Runtime/__init_cpp_exceptions.cpp:
  #.text: [0x802b2c60, 0x802b2cd0]
  #.sdata: [0x8050cfa0, 0x8050cfa8]

Shade/std/shd_cstring.cpp:
  .text: [0x803346e0, 0x803347d0]

Shade/std/shd_stdarg.cpp:
  .text: [0x803347d0, 0x8033496c]